src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+
+  &__image-imac {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    display: block;
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    width: 166px;
```

Comment:
Avoid using fixed width for text elements; it can lead to issues with text not fitting the container on different screen sizes or after text content changes. Use CSS properties like `max-width` along with `word-wrap` or `overflow` to handle text overflow.